src/calculateRentalCost.js

```diff
@@ -29,6 +29,15 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const rentDay = 40;
+
+  if (days >= 1 && days < 3) {
+    return rentDay * days;
+  } else if (days >= 3 && days < 7) {
```

Comment:
```suggestion
  }
  
  if (days >= 3 && days < 7) {
```
Учитывая что в предыдущем if написано `return` - можно else не писать

Reply:
Выдает ошибку lintera, оставила с else